traffic_monitor/datareq/crstate.go

```diff
@@ -21,8 +21,10 @@ package datareq
 
 import (
 	"fmt"
+	"github.com/apache/trafficcontrol/traffic_monitor/todata"
```

Comment:
Nit: Any import starting with `"github.com/apache/trafficcontrol` should be grouped with the other imports that start with `"github.com/apache/trafficcontrol`.